examples/ConsoleApp/ConsoleApp.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net6.0;</TargetFramework>
```

Comment:
what's the extra simcolon?

Reply:
It's pointless. Just added it so they all matched. Can take it out if you prefer 😄

Reply:
Yes, please remove.